app/javascript/components/content/ExampleSpecies.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="cards--example-species">
+    <h3 class="cards__title">{{ content.title }}</h3>
+    
+    <div class="cards__cards">
+      <div 
+        v-for="example in examples" 
```

Comment:
when the linter is merged it will complain. You can use getVForKey from mixin-ids

Reply:
@stacytalbot Did you import the mixin? Also linter will want brackets around example, index